resources/js/processes/designer/AssetButtons.vue

```diff
@@ -1,10 +1,16 @@
 <template>
   <b-card-text>
     <div class="button-container">
-      <slot
-        name="select-template-slot"
-        class="custom-slot"
-      ></slot>
+      <b-button
+        variant="primary"
```

Comment:
remove this line, the color is set in the CSS

Reply:
Solved